src/mxarray.jl

```diff
@@ -3,15 +3,15 @@
 type MxArray
     ptr::Ptr{Void}
     own::Bool
-    
+
     function MxArray(p::Ptr{Void}, own::Bool)
         p == C_NULL && error("NULL pointer for MxArray.")
         self = new(p, own)
         if own
             finalizer(self, release)
         end
         return self
-    end    
+    end
```

Comment:
> ? what do you mean with `w=0`.

https://github.com/JuliaInterop/MATLAB.jl/pull/77/files?w=0

> I just trimmed trailing spaces here.

Right. And don't do that in a PR that changes other things at the same time. It makes reviewing much harder and I almost missed the `transcode` below.